src/coredump/coredumpctl.c

```diff
@@ -180,6 +193,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "no-legend",    no_argument,       NULL, ARG_NO_LEGEND },
                 { "output",       required_argument, NULL, 'o'           },
                 { "field",        required_argument, NULL, 'F'           },
+                { "debugger",     required_argument, NULL, 'd'           },
```

Comment:
Don't use a single-letter option for this at all. It seems to be a fairly niche usecase, and we have a limited supply of single letters. Please just provide a long option for now (like "no-legend" above).